doc/md/tutorial-todo-gql.md

```diff
@@ -261,7 +261,7 @@ func main() {
 	}
 
 	// Configure the server and start listening on :8081.
-	srv := handler.NewDefaultServer(NewSchema(client))
+	srv := handler.NewDefaultServer(todo.NewSchema(client))
```

Comment:
Thanks for fixing this @isoppp.

I think the `"todo"` import is also necessary. Can you add it to this PR? 🙏

Reply:
Ah, thank you for pointing it out, it's also necessary.
Fixed it 🙏